### DISCUSSION TITLE
Improve touch experience (especially with sliders)

### DISCUSSION THREAD

**Discussion by Valkirie** [wherewhere/ModernWpf#24] [General]
Hey, I've been using ModernWpf for quite sometimes now and as much as I love it, I've been facing several issues/discomfort when using Sliders and a touch screen.

1. They're hard to reach and requires the users to precisely touch the thumb for it to record inputs.
2. Sliding a slider within a scroller will also scroll the page.

Do you have ideas on how to improve both the above ?

**Comment by wherewhere** [comment]
Mybe you should move to WAS...

**Reply by Valkirie** [reply]
What is WAS ?

**Reply by wherewhere** [reply]
Windows App SDK
